src/main/kotlin/test/ktortemplate/Application.kt

```diff
@@ -57,6 +58,17 @@ fun Application.module() {
         method(HttpMethod.Post)
         method(HttpMethod.Put)
         method(HttpMethod.Delete)
+        exposeHeader(PaginationHeader.HEADER_PAGE)
```

Comment:
We need to expose this headers so that browsers can read them.

Reply:
As these grow in size, might be better to use an iterator:
```kotlin
listOf( ... ).forEach { exposeHeader(it) }
```